docs/feature_audio.md

```diff
@@ -1,21 +1,45 @@
 # Audio
 
-Your keyboard can make sounds! If you've got a Planck, Preonic, or basically any AVR keyboard that allows access to certain PWM-capable pins, you can hook up a simple speaker and make it beep. You can use those beeps to indicate layer transitions, modifiers, special keys, or just to play some funky 8bit tunes.
+Your keyboard can make sounds! If you've got a Planck, Preonic, or basically any keyboard that allows access to certain PWM-capable pins, you can hook up a simple speaker and make it beep. You can use those beeps to indicate layer transitions, modifiers, special keys, or just to play some funky 8bit tunes.
 
+To activate this feature, add `AUDIO_ENABLE = yes` to your `rules.mk`.
+
+## AVR based boards
 Up to two simultaneous audio voices are supported, one driven by timer 1 and another driven by timer 3.  The following pins can be defined as audio outputs in config.h:
 
-Timer 1:
+the primary voice, with Timer 3 on one of these pins:
+`#define C4_AUDIO`
+`#define C5_AUDIO`
+`#define C6_AUDIO`
+
+and *optionally*, a secondary voice, using Timer 1, on one of these pins:
 `#define B5_AUDIO`
 `#define B6_AUDIO`
 `#define B7_AUDIO`
 
-Timer 3:
-`#define C4_AUDIO`
-`#define C5_AUDIO`
-`#define C6_AUDIO`
 
-If you add `AUDIO_ENABLE = yes` to your `rules.mk`, there's a couple different sounds that will automatically be enabled without any other configuration:
+## ARM based boards
+STM32F2 upwards can use the builtin DAC unit, to drive Pins A4 or A5:
+add `AUDIO_DRIVER = dac` to `rules.mk` and set in `config.h` either:
+`#define A4_AUDIO`
+OR
+`#define A5_AUDIO`
+
+
+STM32F1xx have to fall back to using PWM (on the up side: with any pin you choose),
+
+either set:
+`AUDIO_SYSTEM = pwm-pin-alternate` in `rules.mk` and
+`#define AUDIO_PIN C13` in `config.h`
+to have the selected pin output a pwm signal, generated from a timer callback (e.g. toggled in software)
+ OR
+`AUDIO_DRIVER PWM_PIN_ALTERNATE` in `rules.mk`
+which will use Timer 1 to directly drive pin PA8 through the PWM hardware (TIM1_CH1 = PA8).
+Should you want to use the pwm-hardware on another pin and timer - be ready to dig into the STM32 datasheet and do the neccessary changes to quantum/audio/audio_arm_pwm.c.
```

Comment:
If there are changes required to `quantum/audio/audio_arm_pwm.c` in order to change the pin, please parameterise the inputs via `#define` in order to allow people to perform those modifications by editing their own `config.h`, rather than the qmk sources.